int32 intg_value
---
int32 intg_us_new
